master/buildbot/config.py

```diff
@@ -816,6 +816,34 @@ def __init__(self, name=None, slavename=None, startSlavenames=None, slavenames=N
 
         self.description = description
 
+        self.customBuildUrls = customBuildUrls or {}
```

Comment:
Either make the param `None` (has the benefit of avoiding errors with code altering default values in runtime) or remove this line